applications/sandbox/source.cpp

```diff
@@ -12,6 +12,9 @@
 
 #include "module/testModule.hpp"
 
+#include <core/scenemanagement/SceneManagerModule.hpp>
+#include <core/scenemanagement/scene.hpp>
```

Comment:
maybe merge the scene manager module with the core module, that already gets automatically reported. Scenes are part of core.